vendor_blueprints/export_card_to_file.py

```diff
@@ -0,0 +1,181 @@
+import json
+import re
+import sys
+import argparse
+import requests
+import urllib.parse
+from pydomo import Domo
+
+
+EXIT_CODE_INVALID_CREDENTIALS = 200
+EXIT_CODE_INVALID_ACCOUNT = 201
+EXIT_CODE_BAD_REQUEST = 202
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--email', dest='client_id', required=True)
+    parser.add_argument('--password', dest='password', required=True)
+    parser.add_argument('--client-id', dest='client_id', required=True)
+    parser.add_argument('--secret-key', dest='secret_key', required=True)
+    parser.add_argument('--domo-instance', dest='domo_instance', required=True)
+    parser.add_argument('--filename', dest='filename', required=True)
```

Comment:
Nitpicky request to make this `file-name` and the resulting variable `file_name`

Reply:
Will do, will rename to dest_file_name to make it more readable and better reflect it's purpose